build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -49,7 +49,7 @@ public string ToString()
         //
 
         // Full versions and package information.
-        public string LatestHostPrerelease => "rc3";
+        public string LatestHostPrerelease => ReleaseSuffix;
```

Comment:
Ah, I thought this was inside the `struct VerInfo`.  I see now it it is in the containing class.